<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileEditor3</class>
 <widget class="QDialog" name="ProfileEditor3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>98</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Profile Editor Gen 3</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelProfile">
     <property name="text">
      <string>Profile Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEditProfile"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="labelTID">
     <property name="text">
      <string>TID</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="TextBox" name="textBoxTID"/>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="pushButtonOkay">
     <property name="text">
      <string>Okay</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelVersion">
     <property name="text">
      <string>Version</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="ComboBox" name="comboBoxVersion">
     <item>
      <property name="text">
       <string>Ruby</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sapphire</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fire Red</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Leaf Green</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Emerald</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>XD</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Colosseum</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="labelSID">
     <property name="text">
      <string>SID</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="TextBox" name="textBoxSID"/>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="pushButtonCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QCheckBox" name="checkBoxDeadBattery">
     <property name="text">
      <string>Dead Battery</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextBox</class>
   <extends>QLineEdit</extends>
   <header>Forms/Controls/TextBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>Forms/Controls/ComboBox.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditProfile</tabstop>
  <tabstop>comboBoxVersion</tabstop>
  <tabstop>textBoxTID</tabstop>
  <tabstop>textBoxSID</tabstop>
  <tabstop>checkBoxDeadBattery</tabstop>
  <tabstop>pushButtonOkay</tabstop>
  <tabstop>pushButtonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>ProfileEditor3</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>452</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
